docs/data-masking-function-list.md

```diff
@@ -0,0 +1,898 @@
+
+### Returns
+
+A string with the selected characters masked by a specified `mask_char` or the default value for that parameter. The function supports multibyte characters in any character set. The character set of the return value is the same as `str`.
+
+An error is reported if  `str` length is an incorrect length.
+
+Returns a NULL if you invoke this function with NULL as the primary argument.
+
+### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT mask_canada_sin('555-555-555');
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +--------------------------------+
+    | mask_canada_sin('555-555-555') |
+    +--------------------------------+
+    | XXX-XXX-XXX                    |
+    +--------------------------------+
+    ```
+
+## mask_iban(str [,mask_char])
+
+Masks an Internal Bank Account Number (IBAN).
+
+### Parameters
+
+| Parameter | Optional | Description | Type |
+| --- | --- | --- | --- |
+| `str` | No | The string to be masked | String |
+| `mask_char` | Yes | Character used for masking | String |
+
+The `str` accepts either of the following:
+
+* No separator symbol
+
+* Groups of four characters. These groups can be separated by a space or any separator character.
+
+The default value for `mask_char` is `*`. The value can be a multibyte character in any character set and may not be same character set as `str`.
+
+### Returns
+
+Returns the masked string. The character set of the result is the same as the character set of `str`.
+
+An error is reported if the `str` length is incorrect.
+
+If the `str` contains a multi-byte character, the `str` is not masked.
+
+Returns NULL if you invoke this function with NULL as the primary argument.
+
+### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT mask_iban('DE27 1002 02003 77495 4156');
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +---------------------------------------------+
+    | mask_iban('DE27 1002 02003 77495 4156')     |
+    +---------------------------------------------+
+    | DE** **** **** **** ****                    |
+    +---------------------------------------------+
+    ```
+
+## mask_inner(str, margin1, margin2 [,mask_char])
+
+Returns the string where a selected inner portion is masked with a substitute character.
+
+### Parameters
+
+| Parameter | Optional | Description | Type |
+| --- | --- | --- | --- |
+| `string` | No | The string to be masked | String |
+| `margin1` | No | The number of characters on the left end of the string to remain unmasked | Integer |
+| `margin2` | No | The number of characters on the right end of the string to remain unmasked | Integer |
+| `mask_char` | Yes | The masking character | String |
+
+The `margin1` value cannot be a negative number. If the value is `0`, then all of the left end characters are unmasked.
```

Comment:
Looks like the correct statement would be `are masked`.
Moreover, I personally find this `If the value is 0` too hard to process. it's better be reworded

Reply:
May be something like
then the the masking range starts from the very first character on the left / the masking range ends at the very last character on the right